### ISSUE TITLE
Gets Stuck on Image Upload

### ISSUE THREAD

**Issue by JJGO** [hedgedoc/hedgedoc#788]
**Description**
Trying to upload an image by either pasting or draging into the editor results in a blurb that says `![Uploading file..._<random_string>]()` and the image upload does not complete

**To Reproduce**

1. Paste Image from Clipboard or Drag and drop from local filesystem
 
**Expected behavior**
Upload should complete and Image should be uploaded and viewable like in the demo.

**Logs**

Relevant part of the logs which seems to indicate an issue with `uploads`

```
2021-01-30T17:03:00.906Z info:  deserializeUser: cb6eb18c-cb29-4c07-aa7d-bc647cdbdb0a
(node:304) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_URL]: Invalid URL: /uploads/
    at onParseError (internal/url.js:257:9)
    at new URL (internal/url.js:333:5)
    at new URL (internal/url.js:330:22)
    at Object.exports.uploadImage (/opt/hedgedoc/lib/web/imageRouter/filesystem.js:28:19)    at /opt/hedgedoc/lib/web/imageRouter/index.js:63:22
(node:304) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 8)
```

**Config**

Here's my `docker-compose.yml`. I use linuxserver ARM  build and a postgres backend. I access the instance as `localhost:3000`.

```
version: '3'    
                                     
services:                   
  database:                
    image: postgres:latest
    container_name: hedgedoc_db
    restart: unless-stopped
    volumes:    
      - /opt/hedgedoc/data:/var/lib/postgresql/data                            
    environment:                    
      - POSTGRES_USER=hedgedoc                                             
      - POSTGRES_DB=hedgedoc
      - POSTGRES_PASSWORD=password                                    
  hedgedoc:                
    image: ghcr.io/linuxserver/hedgedoc:latest                                 
    container_name: hedgedoc
    restart: unless-stopped                                                    
    security_opt:                                                              
      - no-new-privileges:true                                                                                        
    depends_on:                                                                
      - database                                                               
    volumes:                                                                   
      - /opt/hedgedoc/config:/config                                           
    ports:
      - 3000:3000                            
    environment:                                                               
      - CMD_DB_URL=postgres://hedgedoc:password@hedgedoc_db:5432/hedgedoc      
      - TZ=America/New_York                                                    
      - PGID=1001                                                              
      - PUID=1001                                                           
```

**Your Setup (please complete the following information):**
 - Host OS: Raspbian GNU/Linux 10
 - NodeJS version: v12.20.1
 - HedgeDoc version (Click "Version info" in the footer).
     - I could not find said footer, Docker reports image is version 1.7.2

**Comment by davidmehren** [comment]
This error indicates that you did not set up `CMD_DOMAIN` and related variables correctly. You should see a corresponding warning on startup.
Please have a look at https://docs.hedgedoc.org/configuration/#hedgedoc-location for more information.

Note that this is also stated in [the linuxserver docs](https://github.com/linuxserver/docker-hedgedoc#application-setup).

**Comment by JJGO** [comment]
Thank you, that was indeed the issue. 

For reference, I had to add the following env vars to the docker-compose setup. These are undocumented on the linuxserver repo, I might pull request some info, so people don't run into the same issue. I also enabled SSL since I'm putting this behind a reverse proxy with HTTPS.

```yml
    environment:
      [...]
      - CMD_DOMAIN=hedgedoc.mydomain.com
      - CMD_PROTOCOL_USESSL=true
```

Thanks again and closing the issue.
